R/sidebar.R

```diff
@@ -199,6 +220,10 @@ sidebar_close <- function(id, session = get_current_session()) {
   session$onFlush(callback, once = TRUE)
 }
 
+collapse_icon <- function() {
+  HTML("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' class='collapse-icon' fill='currentColor'><path fill-rule='evenodd' d='M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z'/></svg>")
+}
```

Comment:
Maybe worth using `bsicons` if available?

Reply:
If we do, maybe we should also be doing that for `card(full_screen = TRUE)`

Reply:
Done in d8e5b0577f1b4f90b425c8b237fd950171168310. For now, I only used bsicons for the collapse icon. I actually like this approach a lot and if we decided to make the collapse and fullscreen icons (more easily) customizable, bsicons would be a nice way to achieve this.